[2025-02-14T22:53:21.335+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-14T22:53:21.357+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: manage_airflow_roles.get_dags_by_owner manual__2025-02-14T22:53:20.504864+00:00 [queued]>
[2025-02-14T22:53:21.373+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: manage_airflow_roles.get_dags_by_owner manual__2025-02-14T22:53:20.504864+00:00 [queued]>
[2025-02-14T22:53:21.373+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-14T22:53:21.392+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): get_dags_by_owner> on 2025-02-14 22:53:20.504864+00:00
[2025-02-14T22:53:21.400+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=411) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-14T22:53:21.403+0000] {standard_task_runner.py:72} INFO - Started process 418 to run task
[2025-02-14T22:53:21.403+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'manage_***_roles', 'get_dags_by_owner', 'manual__2025-02-14T22:53:20.504864+00:00', '--job-id', '327', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp7e9qnrwb']
[2025-02-14T22:53:21.406+0000] {standard_task_runner.py:105} INFO - Job 327: Subtask get_dags_by_owner
[2025-02-14T22:53:21.468+0000] {task_command.py:467} INFO - Running <TaskInstance: manage_airflow_roles.get_dags_by_owner manual__2025-02-14T22:53:20.504864+00:00 [running]> on host 71404c6b5320
[2025-02-14T22:53:21.579+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data-team' AIRFLOW_CTX_DAG_ID='manage_***_roles' AIRFLOW_CTX_TASK_ID='get_dags_by_owner' AIRFLOW_CTX_EXECUTION_DATE='2025-02-14T22:53:20.504864+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-14T22:53:20.504864+00:00'
[2025-02-14T22:53:21.580+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-14T22:53:21.583+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/***/dags
[2025-02-14T22:53:21.599+0000] {python.py:240} INFO - Done. Returned value was: {'dados': ['hello_world_dag'], 'data-team': ['manage_***_roles']}
[2025-02-14T22:53:21.629+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-14T22:53:21.630+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=manage_***_roles, task_id=get_dags_by_owner, run_id=manual__2025-02-14T22:53:20.504864+00:00, execution_date=20250214T225320, start_date=20250214T225321, end_date=20250214T225321
[2025-02-14T22:53:21.659+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-14T22:53:21.672+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
